nodejsVersions.md

```diff
@@ -0,0 +1,18 @@
+<!--
```

Comment:
The root doesn't seem like the right place for this. Maybe in the layouts or https://github.com/nodejs/nodejs.org/tree/master/layouts/partials

Reply:
@nschonni: Thanks for the idea, maybe this is a mistake by you. What I aim is to just notify the user the usage of the md file instead of saying about layouts...What's more: I cannot find a suitable place to put it in the doc.hbs? Or do you think where to put? Or remove it as you think it's useless?

Reply:
I'll remove it.

Reply:
Sorry, my comment wasn't about the comments at the top of the file, it was that this whole file doesn't belong in the root of the project and should be in the layouts

Reply:
What about putting into the 'source'? I notice that there're many data from 'source' folder, such as 'schedule.json', 'versions.json'?

Reply:
Oh thanks! You reguard it as a template file, that's fine with me ;)

Reply:
Oh thanks! You reguard it as a template file, that's fine with me ;)

Reply:
Oh thanks! You reguard it as a template file, that's fine with me ;)